glossary.md

```diff
@@ -0,0 +1,10 @@
+# Glossary
+**Asset owner**: A human that owns digital assets and wishes to store related secrets using Lock Keeper. 
+
+**Demo repository**: A repository that demonstrates usage of the client and server libraries. <br>
+[TODO](https://github.com/boltlabs-inc/key-mgmt/issues/93): Add location after set-up.
+
+**Client**: Software that runs on an asset owner's device and initiates workflows involving secrets.
```

Comment:
Should we separately define the local and remote clients, or is this saying in our current PoC, there is just this definition of "client"?

Reply:
This is just referring the current PoC, yes. I figured it's less confusing if we iteratively build out the design, phase by phase.